test/save-in-progress/01-sip-finish-later.e2e.spec.js

```diff
@@ -79,13 +79,15 @@ module.exports = E2eHelpers.createE2eTest(
       .click('.schemaform-sip-save-link');
     /* eslint-enable camelcase */
 
+    client.waitForElementVisible('.saved-form-metadata-container', Timeouts.normal);
     client.assert.urlContains('form-saved');
```

Comment:
Do you think it would it make any difference to change that to `Timeouts.slow`?

Reply:
I can try